### PR TITLE
Align comments between netfx and netcore codebases

### PR DESCRIPTION
**Pull request by MichelZ**
Hopefully an easy to review PR that deals with differences in comments and some whitespace between netfx and netcore for easier future merging

Part of #2965

### CODE REVIEW

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs [hunk 1]
```diff
@@ -2045,6 +2046,7 @@ override public TextReader GetTextReader(int i)
         }
 
         /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlDataReader.xml' path='docs/members[@name="SqlDataReader"]/GetChar/*' />
+        [EditorBrowsableAttribute(EditorBrowsableState.Never)]
```
Comment (benrr101) [inline src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs:2049]: I'm not sure that this should be moved across to netcore.

**Review by benrr101 — CHANGES_REQUESTED** [review]
Please rollback the change for `EditorBrowsableAttribute` if we don't have a strong reason to move it over.

Also, in the future, please just combine all the comment cleanup changes into a single PR.

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs (hunk 1 above)
Comment (MichelZ) [reply src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs:2049]: For alignment purpuses, is it OK to remove it from netfx then?

Comment (edwardneal) [reply src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs:2049]: I've just checked the ref projects; this is present in both [netfx](https://github.com/dotnet/SqlClient/blob/main/src/Microsoft.Data.SqlClient/netfx/ref/Microsoft.Data.SqlClient.cs#L1294) and [netcore](https://github.com/dotnet/SqlClient/blob/main/src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.cs#L1282). I personally think it's more likely that someone missed a spot when adding the attribute before the source code was uploaded. In light of the ref projects, perhaps we should add the EditorBrowsableAttribute to netcore?

Comment (MichelZ) [reply src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs:2049]: > I think this is the sort of thing that for merging purposes we should leave alone. In the merged version, wrap it in a `#if NETFRAMEWORK`. While it's not the cleanest solution, we have precedent for doing this in other merges, and it is the safest change.

What about a combination, wrapping it in a #IF NETFRAMEWORK in both versions?

Comment (benrr101) [reply src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs:2049]: I think if it's present in both ref projects, we should probably have it in both netfx and netcore implementation projects. I officially rescind my comment :)

**Review by benrr101 — APPROVED** [review]

**Review by mdaigle — APPROVED** [review]

**Pull request merged**